README.md

```
@@ -17,43 +17,20 @@ This repository is a global conversion mod for the game Space Station 14. Nearly
-```
-#### 4. Обновляем подмодули сборки
-Вводим в консоль:
-```
-git submodule update --init --recursive
-```
-#### 5. Собираем сборку
-Вводим в консоль:
+## Building
 
+1. Clone this repo:
+```shell
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)